.travis.yml

```diff
@@ -32,6 +32,7 @@ r_github_packages:
   - berndbischl/parallelMap
   - berndbischl/ParamHelpers
   - RevolutionAnalytics/quickcheck/pkg
+  - ropensci/plotly
```

Comment:
@larskotthoff 
Could you help me to check if it is right to add a github packages?